fineract-provider/build.gradle

```diff
@@ -42,6 +42,7 @@ buildscript {
      classpath "org.apache.openjpa:openjpa:$openJPAVersion"
      classpath 'com.radcortez.gradle:openjpa-gradle-plugin:3.1.0'
      classpath 'gradle.plugin.org.nosphere.apache:creadur-rat-gradle:0.2.2'
+      classpath "gradle.plugin.org.hidetake:gradle-swagger-generator-plugin:2.18.1"
```

Comment:
I think this plugin would not help to generate the swagger.json automatically during project build
please remove this.